pandas/tests/frame/indexing/test_getitem.py

```diff
@@ -67,6 +67,12 @@ def test_getitem_sparse_column_return_type_and_dtype(self):
         result = df.loc[:, "A"]
         tm.assert_series_equal(result, expected)
 
+    def test_getitem_indexer_empty_series(self):
+        # GH: 19290
+        df = DataFrame(columns=["a", "b", "c"])
+        result = df[Series([], name="a", dtype="object")]
```

Comment:
does the dtype of teh series matter here? e.g. is bool ok?

Reply:
Bool is ok, but had to change the if condition for empty series